client/src/commponet/Description.js

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+// import "bootstrap/dist/css/bootstrap.min.css";
+import { Container, Form, Row, Col } from "react-bootstrap";
```

Comment:
Again - I think you need to discuss in the team which frontend libraries you are going to use?